internal/users/manager.go

```diff
@@ -212,6 +226,42 @@ func (m *Manager) UpdateUser(u types.UserInfo) (err error) {
 	return nil
 }
 
+// checkGroupNameConflict checks if a group with the given name already exists.
+// If it does, it checks if it has the same UGID.
+func (m *Manager) checkGroupNameConflict(name string, ugid string) error {
+	// Check if a group with that name already exists. First check in our database.
```

Comment:
That's what the function does, so can simplify:
```suggestion
	// First check in our database.
```

Reply:
done